build_manifest.json

```diff
@@ -53,8 +53,15 @@
     "rebuildPatterns": [
       "^l1-contracts/"
     ],
+    "dependencies": []
+  },
+  "l1-artifacts": {
+    "buildDir": "l1-artifacts",
+    "dockerfile": "Dockerfile",
+    "rebuildPatterns": [
+      "^l1-artifacts/"
+    ],
     "dependencies": [
-      "ethereum.js",
       "foundation"
     ]
```

Comment:
Shouldn't l1-artifacts depend on l1-contracts, so they get regenerated whenever the solidity code changes?

Reply:
The should. Good catch.